content/applications/marketing/surveys/live_session.rst

```diff
@@ -0,0 +1,218 @@
+====================
+Live Session surveys
+====================
+
+With the Odoo *Surveys* application, users can enhance in-person demonstrations and presentations
+with the *Live Session* feature.
+
+*Live Session* surveys function the same way as a normal survey, but with a host (or moderator), who
+presents the questions to participants, reveals their responses in real-time, and controls the tempo
+of the overall survey.
+
+In *Live Session* surveys, participants access the survey experience via a custom URL, and sign-in
+with an optional access code. When the survey has begun, the host presents one question at a time.
+
+Then, the audience of participants submit their answer, either via their computer or mobile device,
+and once the resposnes have been gathered, the host reveals all the participant's responses, in
+real-time, with each answer's results displayed as a bar graph.
+
+.. image:: live_session/live-session-concept-sample.png
+   :align: center
+   :alt: Rendered concept of how a Live Session question and answer result appears in Odoo Surveys.
+
+Create Live Session survey
+==========================
+
+To create a live session survey, begin by opening the *Surveys* application. From the
```

Comment:
```suggestion
To create a *Live Session* survey, begin by opening the *Surveys* application. From the
```
Should this be italicized and capitalized like the previous instances?

Reply:
I see that it is not italicized or capitalized from here on out, so I won't add a bunch of comments to anything following this one - I'll leave this one in case it should be changed though